core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.scss

```diff
@@ -30,3 +26,7 @@
   cursor: pointer;
   color: #007bff;
 }
+
+::ng-deep .cdk-overlay-container {
```

Comment:
Can you please add a comment to explain what this line is for and why it is needed for future coders to understand.

Reply:
done